### PR TITLE
Update prod-deploy.yml

### PR DESCRIPTION
**Pull request by GoliathLabs**
### Description

I've added the GH-Cache setting for the build-push action. This should speed up the build process.

Fixes #

#### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

Preview: https://62f2df6c8a2c4f25bef1bdcc--pr-cinny.netlify.app
⚠️ Exercise caution. Use test accounts. ⚠️

### CODE REVIEW

**Comment by GoliathLabs** [comment]
I have also introduced the arm v7 target for the image (for those using the image on an older SoC)

File: .github/workflows/prod-deploy.yml [hunk 1]
```diff
@@ -89,7 +89,9 @@ jobs:
         uses: docker/build-push-action@v3.1.1
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```
Comment (kfiven) [inline .github/workflows/prod-deploy.yml]: It fails at installing dependencies in case of `linux/arm/v7`
```
 > [linux/arm/v7 builder 4/6] RUN npm ci:
#26 273.0 npm ERR! code 1
#26 273.0 npm ERR! path /src/node_modules/sharp
#26 273.0 npm ERR! command failed
#26 273.0 npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
#26 273.0 npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.6/libvips-8.10.6-linuxmusl-armv7.tar.br
#26 273.0 npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
#26 273.0 npm ERR! sharp: Installation error: Prebuilt libvips 8.10.6 binaries are not yet available for linuxmusl-armv7
```

File: .github/workflows/prod-deploy.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3.4.1
         with:
-          node-version: 17.9.0
+          node-version: 17.9.1
```
Comment (kfiven) [inline .github/workflows/prod-deploy.yml]: Why this change?

**Review by kfiven — CHANGES_REQUESTED** [review]

**Comment by kfiven** [comment]
I have tested both the PRs and the build time is the same https://github.com/kfiven/cinny/actions/runs/2831366316
Wondering if we are doing it correct.

**Comment by GoliathLabs** [comment]
According to https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#github-cache this should be correct. But it also depends on what parts can be cached by Github. It might make sense to cache already at the PR level, so that we can mostly reuse the build from the PR when merging into the dev branch
